tiledb/sm/array_schema/test/unit_domain.cc

```diff
@@ -36,18 +36,27 @@
 #include "tiledb/sm/enums/datatype.h"
 #include "tiledb/sm/enums/layout.h"
 
+#include "tiledb/api/c_api/context/context_api_internal.h"
```

Comment:
This might cause the build issues?

Reply:
The issue was that the helpers header includes the CPP header which contains some deprecated declarations.

Reply:
We shouldn't add this header to a unit test.